packages/mds-types/transformers/0_4_1_to_1_0_0/index.ts

```diff
@@ -0,0 +1,114 @@
+import { VEHICLE_REASON_0_4_1, VehicleEvent_v0_4_1, VEHICLE_EVENT_0_4_1 } from '../@types'
+import { VEHICLE_EVENT, VehicleEvent, VEHICLE_STATE } from '../../index'
+
+type EVENT_TYPE_REASONS = VEHICLE_REASON_0_4_1 | 'no_event_type_reason'
+
+export const FULL_STATE_MAPPING_0_4_1_to_1_0_0: {
+  [P in VEHICLE_EVENT_0_4_1]: {
+    [Q in EVENT_TYPE_REASONS]: {
+      event_type: VEHICLE_EVENT
+      vehicle_state: VEHICLE_STATE
```

Comment:
I think we ought to have `VEHICLE_EVENT_1_0_0`, `VEHICLE_STATE_1_0_0`, etc. to use in the transformers. Then we should create aliases `type VEHICLE_EVENT = VEHICLE_EVENT_1_0_0` ... this might eliminate changes if we have a similar exercise for 1.1, 2.0, etc.

Reply:
Adding on to this, now that we have the aliases we likely want to make it so that the mappings here are explicitly to 1.0.0, and not to the aliases. So 
```suggestion
      event_type: VEHICLE_EVENT_1_0_0
      vehicle_state: VEHICLE_STATE_1_0_0
```

Reply:
Yeah sorry, I ended up doing that in the device_transformers branch.